dist/robotframework/Makefile.include

```diff
@@ -1,11 +1,10 @@
 # RF specific paths
 RFBASE    ?= $(TESTBASE)/dist/robotframework
 RFPYPATH  ?= $(APPDIR)/tests:$(RFBASE)/lib:$(RFBASE)/res
-RFOUTPATH ?= $(BUILD_DIR)/robot/$(BOARD)/$(APPLICATION)/
-# search for RF test script files
-ROBOT_FILES ?= $(sort $(wildcard tests/*.robot))
+RFOUTPATH ?= $(BUILD_DIR)/robot/$(BOARD)/$(APPLICATION)
+ROBOT_TESTDIR ?= tests/
 
-$(RFOUTPATH)/output.xml: $(ROBOT_FILES)
+$(RFOUTPATH)/output.xml: $(ROBOT_TESTDIR)
 	python3 -m robot.run --noncritical skip \
```

Comment:
```suggestion
	-python3 -m robot.run --noncritical skip \
```
So that if an error occurs it ignores it and still tries to run the xunit script